package.json

```diff
@@ -17,16 +17,11 @@
         "Programming Languages"
     ],
     "activationEvents": [
-        "onLanguage:starlark",
```

Comment:
Let's keep `onLanguage:starlark`, on the off chance case someone opens a `.bzl` file in a workspace without any of the other `workspaceContains` triggers. Doesn't hurt.

Reply:
According to [VS Code's documentation](https://code.visualstudio.com/api/references/activation-events#onLanguage), the `onLanguage:starlark` activation event is unnecessary:

> Beginning with VS Code 1.74.0, languages contributed by your extension do not require a corresponding onLanguage activation event declaration for your extension to be activated.

Note that this extension only support VSCode >= 1.85.0

Reply:
I updated the description of this PR to also include the reasoning on why I removed all pre-existing activation events

Reply:
Got it, thanks, sounds good!